juju/juju
# Remove the disable-raft feature flag

## Description of change

This functionality is replaced by legacy-leases, but we always require the raft workers running. We're not aware of anyone using the disable-raft flag to turn off the raft workers in 2.4

## QA steps

* bootstrap a 2.4 controller with disable-raft, verify that the workers aren't running.
* upgrade to 2.5 - see that the raft workers are running.
